tests/thread_queue/thread_queue.c

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <errno.h>
+#include <stdlib.h>
```

Comment:
Are there LTP tests that can be run for FUTEX?

Reply:
`/ltp/testcases/kernel/syscalls/futex/futex_wait_bitset01` tests bitset directly but is hanging, I'm still investigating.